SimPlatform/dashboard.h

```diff
@@ -0,0 +1,25 @@
+#ifndef DASHBOARD_H
+#define DASHBOARD_H
+
+#include <QMainWindow>
+
+namespace Ui {
+class DashBoard;
+}
+
```

Comment:
The main view should contain more information than that. See #2 for a first list of all of the data it should contain. All other views of the UI will extend this view.